### ISSUE TITLE
Update flutter_secure_storage version

### ISSUE THREAD

**Issue by malikdoksoz** [darwin-morocho/flutter-facebook-auth#315]
### What version are you using?

flutter_facebook_auth: 5.0.6

### What OS and version are you using to local deploy your application?

macOs 12.6.1

### What platforms are you seeing the problem on?

iOS

### pubspec.yaml

```yaml
name: app
description: app

# The following line prevents the package from being accidentally published to
# pub.dev using `flutter pub publish`. This is preferred for private packages.
publish_to: "none" # Remove this line if you wish to publish to pub.dev

version: 1.0.0+4

environment:
  sdk: ">=2.17.1 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  cupertino_icons: ^1.0.2
  # Database
  hive: ^2.2.2
  hive_flutter: ^1.1.0
  flutter_secure_storage: ^7.0.0
  flutter_dotenv: ^5.0.2
  #State Managament
  hooks_riverpod: ^1.0.4
  introduction_screen: ^3.1.1
  flutter_svg: ^1.1.6
  confetti: ^0.7.0
  permission_handler: ^10.2.0
  styled_text: ^6.0.0
  easy_localization: ^3.0.1
  easy_logger: ^0.0.2
  carousel_slider: ^4.2.0
  flutter_form_builder: ^7.7.0
  flutter_chat_ui: ^1.6.6
  uuid: ^3.0.7
  device_info_plus: ^8.0.0
  firebase_messaging: ^14.1.4
  # sign in providers
  flutter_facebook_auth: ^5.0.6
```

### Describe the Bug

Running "flutter pub get" in app...                    
Because flutter_facebook_auth >=5.0.6 depends on facebook_auth_desktop ^0.0.7 which depends on flutter_secure_storage ^6.0.0, flutter_facebook_auth >=5.0.6 requires flutter_secure_storage ^6.0.0.
So, because app depends on both flutter_secure_storage ^7.0.0 and flutter_facebook_auth ^5.0.6, version solving failed.

### Expected Behavior

flutter_secure_storage dependency in **facebook_auth_desktop** library needs to be upgraded to version 7.0.0

### To Reproduce

...

### flutter doctor -v

```shell
[✓] Flutter (Channel stable, 3.3.9, on macOS 12.6.1 21G217 darwin-x64, locale tr-TR)
    • Flutter version 3.3.9 on channel stable at /Users/malikdoksoz/TumDosyalar/Dosyalar/flutter_src/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b8f7f1f986 (2 weeks ago), 2022-11-23 06:43:51 +0900
    • Engine revision 8f2221fbef
    • Dart version 2.18.5
    • DevTools version 2.15.0

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/malikdoksoz/Library/Android/sdk
    • Platform android-33, build-tools 30.0.3
    • ANDROID_HOME = /Users/malikdoksoz/Library/Android/sdk
    • ANDROID_SDK_ROOT = /Users/malikdoksoz/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 14B47b
    • CocoaPods version 1.11.3

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.74.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.54.0

[✓] Connected device (2 available)
    • iPhone 7 Plus (mobile) • 64c5a3f0568bf94d82527243e5ca87a1cf3ee18b • ios        • iOS 15.7.1 19H117
    • macOS (desktop)        • macos                                    • darwin-x64 • macOS 12.6.1 21G217
      darwin-x64
    ! Error: iPhone 7 Plus is busy: Preparing iPhone 7 Plus for development. Xcode will continue when iPhone 7
      Plus is finished. (code -10)

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by darwin-morocho** [comment]
Fixed on 5.0.7

**Comment by cnkygmr** [comment]
Because no versions of flutter_facebook_auth match >5.0.7 <6.0.0 and flutter_facebook_auth 5.0.7 depends on facebook_auth_desktop ^0.0.8, flutter_facebook_auth ^5.0.7 requires facebook_auth_desktop ^0.0.8.
Because facebook_auth_desktop 0.0.8 depends on flutter_secure_storage ^7.0.1 and no versions of facebook_auth_desktop match >0.0.8 <0.1.0, facebook_auth_desktop ^0.0.8 requires flutter_secure_storage ^7.0.1.
Thus, flutter_facebook_auth ^5.0.7 requires flutter_secure_storage ^7.0.1.
So, because xxx depends on both flutter_facebook_auth ^5.0.7 and flutter_secure_storage ^8.0.0, version solving failed.
pub get failed

**Comment by cnkygmr** [comment]
@darwin-morocho 

Because no versions of flutter_facebook_auth match >5.0.7 <6.0.0 and flutter_facebook_auth 5.0.7 depends on facebook_auth_desktop ^0.0.8, flutter_facebook_auth ^5.0.7 requires facebook_auth_desktop ^0.0.8.
Because facebook_auth_desktop 0.0.8 depends on flutter_secure_storage ^7.0.1 and no versions of facebook_auth_desktop match >0.0.8 <0.1.0, facebook_auth_desktop ^0.0.8 requires flutter_secure_storage ^7.0.1.
Thus, flutter_facebook_auth ^5.0.7 requires flutter_secure_storage ^7.0.1.
So, because xxx depends on both flutter_facebook_auth ^5.0.7 and flutter_secure_storage ^8.0.0, version solving failed.
pub get failed

**Comment by darwin-morocho** [comment]
> flutter_secure_storage

fixed on 5.0.8
